src/view/com/composer/ExternalEmbedRemoveBtn.tsx

```diff
@@ -11,7 +11,7 @@ export function ExternalEmbedRemoveBtn({onRemove}: {onRemove: () => void}) {
   const {_} = useLingui()
 
   return (
-    <View style={[a.absolute, a.pt_sm, a.pr_sm, {top: 0, right: 0}]}>
+    <View style={[a.absolute, a.pt_sm, a.pr_sm, a.z_50, {top: 0, right: 0}]}>
```

Comment:
this seems like it would be better fixed at the usage site, no? we can't guess the necessary z-order here, but we can be more certain in the place that defines the actual ordering (and it's possible we can also fix this by changing the DOM order itself)